catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -142,10 +142,10 @@ private void registerBasicMetacard() {
     Bundle bundle = FrameworkUtil.getBundle(CatalogFrameworkImpl.class);
     if (bundle != null && bundle.getBundleContext() != null) {
       Dictionary<String, Object> properties = new DictionaryMap<>();
-      properties.put("name", BasicTypes.BASIC_METACARD.getName());
```

Comment:
❓ Not sure I understand how we were using BasicTypes from `catalog-core-api-impl` if it was not embedded in this bundle, but now we have to embed `catalog-core-api-impl` to use `MetacardImpl`, but they are both in the same jar?